pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -361,6 +361,14 @@ func aclToRules(aclArgs types.AppNetworkACLArgs, ACLs []types.ACE) (types.IPTabl
 				"-p", "tcp", "--sport", "domain"}
 			aclRule4.Action = []string{"-j", "ACCEPT"}
 			rulesList = append(rulesList, aclRule1, aclRule2, aclRule3, aclRule4)
+
+			aclRule1.Rule = []string{"-i", aclArgs.BridgeName, "-d", "169.254.169.254",
```

Comment:
PR description says that the meta-data server is run on bridge IP address. Code seems to still use 169.254.169.254
Did I miss something here?

Reply:
The caller in the dom0 connects to the standard 169.254.169.254, but the server is deployed on the bridgeIP. NAT makes the glue.